scripts/deploy/deploy.js

```diff
@@ -48,7 +48,7 @@ const productTypes = [
     productType: {
       descriptionIpfsHash: 'yieldTokenCoverIPFSHash',
       claimMethod: claimMethod.incident,
-      gracePeriodInDays: 14,
```

Comment:
@mixplore I think the new `gracePeriod` param is now in seconds, not in days anymore - @shark0der ?